docs/reference/commandline/secret_create.md

```diff
@@ -34,32 +34,32 @@ command on a manager node.
 ### Create a secret
 
 ```bash
-$ cat secret.json | docker secret create -f - secret.json
+$ echo <secret> | docker secret create -f - my_secret
```

Comment:
@thaJeztah I understand what it is doing, I just did not realize that `-f` would be a required flag even when piping from STDIN. Do other Docker commands work like that?